.github/workflows/master.yml

```diff
@@ -52,27 +52,33 @@ jobs:
     - name: Install system dependencies
       run: |
         sudo apt-get install -y libcurl4-openssl-dev
-    - name: Install dependencies
+    - name: Install R dependencies
+      shell: Rscript {0}
       run: |
         install.packages("devtools")
         remotes::install_deps('mlflow/R/mlflow', dependencies = TRUE, upgrade = FALSE)
-      shell: Rscript {0}
-    - name: Run tests
+    - name: Build package
+      working-directory: mlflow/R/mlflow
       run: |
-        source ./dev/install-common-deps.sh
-        cd mlflow/R/mlflow
         R CMD build .
-        export LINTR_COMMENT_BOT=false
-        cd tests
+    - name: Create test env
+      working-directory: mlflow/R/mlflow/tests
+      run: |
+        Rscript ../.create-test-env.R
+    - name: Run tests
+      env:
+        LINTR_COMMENT_BOT: false
+      working-directory: mlflow/R/mlflow/tests
+      run: |
         Rscript ../.run-tests.R
```

Comment:
I think it would be nice to run this from the current working directory. Currently, we set the directory to tests to run a script in the R package root. Can we simplify? Would need adaption of `.run-tests.R` to account for different current working directory.